api/scripts/migrate

```diff
@@ -0,0 +1,2 @@
+#
+sudo docker run -e DATABASE_URL=<DB_URL> greaterthanfinance/cobudget-api-stage rake db:migrate
```

Comment:
This looks like it needs attention. Other files in this folder look like : 

```
#! /bin/bash
cd /cobudget-api; env -i $(cat /root/env-vars) rake cobudget:deliver_recent_activity_emails
```

Things I notice that are different here:
- there's a missing `#! /bin/bash`
- this script seems to be locked to a staging api (cobudget-api-stage)?
  - I've not read about what docker is doing in this project, so could be off

Reply:
@chime-mu Is this part of the server setup changes? Is this line supposed to be removed?

Reply:
@mixmix @jlopker 
* It's safe to add `#! /bin/bash` to the `migrate` script - but hardly necessary. The script is currently more of a reminder for me (that I wanted to check in) about how to do a migration when everything is running in a docker container. 
* Yes, this script is for update using the stage api docker image, but can be used for a production image as well. Again, it's a work in progress and will be consolidated into a consistent documented deployment pipeline.

Reply:
The `activity_emals` script is a script to be executed by `cron` so it won't function unless the executable is specified. Hence `#! /bin/bash`